test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -13091,7 +13091,7 @@ class M(torch.nn.Module):
             def __init__(self):
                 super().__init__()
                 self.quant = torch.ao.quantization.QuantStub()
-                self.conv = torch.nn.Conv2d(2, 4, 3, stride=2)
+                self.conv = torch.nn.Conv2d(4, 2, 3, stride=2)
```

Comment:
See my comment about these typos. If you expand the diff, the weights are replaced by:
```
        model.conv.weight = torch.nn.Parameter(
            _construct_tensor_for_quantization_test((2, 4, 3, 3), max_val=2)
        )
```
But this is not consistent with `Conv2d(2, 4, 3)`, it is consistent with `Conv2d(4, 2, 3)`, which is what I'm changing it to.